controllers/admin/AdminPPreferencesController.php

```diff
@@ -225,6 +225,14 @@ public function __construct()
                         'required'   => false,
                         'type'       => 'bool',
                     ],
+                    'PS_SHOW_CONDITION'             => [
+                        'title'      => $this->l('Show condition'),
+                        'hint'       => $this->l('Show/Hide conditions dropdown list on product page.'),
```

Comment:
Sorry, my bad. As I don't have no access here please change this hint to
*Show/Hide condition  on product page.*